fundraiser/project/Forc.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 members = [
   "./contracts/fundraiser-contract",
-  "./contracts/fundraiser-contract/tests/artifacts/asset",
+  "./contracts/test-artifacts/asset",
```

Comment:
As mentioned in other PRs, this should either be put back or removed and replaced with functionality from the SDK

Reply:
This is required due to changes with nested projects and forc workspaces. An error is produced if the test artifacts are kept in their old places.

Reply:
I've re-written to use the SDK